@here/harp-mapview/lib/ClipPlanesEvaluator.ts

```diff
@@ -684,19 +692,18 @@ export class TiltViewClipPlanesEvaluator extends TopViewClipPlanesEvaluator {
             z2 = cameraAltitude + sinBeta * cam.top;
             z1 = cameraAltitude - sinBeta * cam.bottom;
         }
-        // Distance along the top plane to the ground - c2
+        // Distance along the top plane to the ground, |c2|. This will determine the far distance,
```

Comment:
`Distance along the top plane to the ground` is someone redundant, |c2| is adequate enough, because it is defined above in the ASCII chart.

Reply:
done